### ISSUE TITLE
Error on plugin startup

### ISSUE THREAD

**Issue by FlyingDiver** [Ghawken/Indigo-EmbyPlugin#2]
```
   Emby Plugin Error               Error in plugin execution startup:

  File "plugin.py", line 147, in startup
type: '>=' not supported between instances of 'str' and 'int'

   Emby Plugin Error               Error in plugin execution runConcurrentThread:

  File "plugin.py", line 127, in runConcurrentThread
type: '>=' not supported between instances of 'str' and 'int'
```

**Comment by Ghawken** [comment]
Interesting - not seeing it on my testing.

I presume this comes from preferences file overriding file type changes in code (when preferences file exists)

Have updated 0.1.4

(this is a very old code base...and just as well the Emby Library c# plugin still working.. which is needed for this)

**Comment by FlyingDiver** [comment]
I haven't looked at the code.  I think Emby now has a pure REST API so you shouldn't need anything C#.

**Comment by FlyingDiver** [comment]
Fixed in 0.1.5.
